README.md

```diff
@@ -1208,6 +1221,79 @@ GET /v3/apps/:guid?fields=guid,name&fields[droplet]=guid
 }
 ```
 
+
+```json
+GET /v3/service_instances?fields[service_plan.service_offering]=name
+
+{
+	"resources": [
+        {
+			"name": "my-managed-instance",
+			"relationships": {
+				"space": {
+					"data": {
+						"guid": "ba0a384b-2b6a-4535-a663-ae018b23f76b",
+						"name": "space-name"
+					}
+				},				
+                "service_plan": {
+					"data": {
+						"guid": "123a384b-2b6a-4535-a663-ae018b23f76b",
+						"name": "plan-one"
+					}
+				}
+			}
+		},
+		{
+			"name": "my-other-managed-instance",
+			"relationships": {
+				"space": {
+					"data": {
+						"guid": "ba0a384b-2b6a-4535-a663-ae018b23f76b",
+						"name": "space-name"
+					}
+				},				
+                "service_plan": {
+					"data": {
+						"guid": "abca384b-2b6a-4535-a663-ae018b23f76b",
+						"name": "plan-two"
+					}
+				}
+			}
+		},
+	],
+	"included": {
+		"service_plans": [
+            {
+                "guid": "123a384b-2b6a-4535-a663-ae018b23f76b"
+                "relationships": {
+                    "service_offering": {
+                        "data": {
+                            "guid": "13c60e38-11e7-11ea-9106-33ee3c5bd4d7"
+                        }
+                    }
+                }
+            },
+            {
+                "guid": "abca384b-2b6a-4535-a663-ae018b23f76b"
+                "relationships": {
+                    "service_offering": {
+                        "data": {
+                            "guid": "13c60e38-11e7-11ea-9106-33ee3c5bd4d7"
+                        }
+                    }
+                }
+            }
+        ],
+		"service_offerings": [
+            {
+                "name": "offering-name"
```

Comment:
If there is a list of multiple service offerings, how will clients link these names back to which object they are related to? For included resources, clients can link them together via guids.

Reply:
Updated to include the guid as well.